src/components/ModelTableList/_model-table-list.scss

```diff
@@ -27,8 +26,6 @@
 }
 
 .models {
-  @extend %row-hover;
```

Comment:
These hover states were left from when you could click anywhere in the row and should have been removed when we changed to only clicking on the names.